### ISSUE TITLE
sitegen.cmd.actions missing

### ISSUE THREAD

**Issue by rfajfar** [leafo/sitegen#6]
I've installed **sitegen** via `sudo luarocks install --server=http://luarocks.org/dev sitegen` and there was no error during the installation.

However when I try to run sitegen it says that the module `sitegen.cmd.actions` can not be found.

Any idea?

**Comment by leafo** [comment]
hmm, this is a problem with development rockspecs, it's easy for them to get out of date with the repo when pointing at master. I updated the one on luarocks.org, but the safest approach would be to pull from github:

```
luarocks install https://raw.githubusercontent.com/leafo/sitegen/master/sitegen-dev-1.rockspec
```

I'll look into a way to synchronize the dev rockspecs from github to luarocks.org

**Comment by rfajfar** [comment]
Using the line above gets me further but it stops at this error:

`moon: /usr/local/share/lua/5.2/sitegen/common.lua:37: /usr/local/share/lua/5.2/moon/init.lua:50: attempt to call global 'getfenv' (a nil value)`

I get this when running `sitegen`. It says `using: site.moon` followed by this error message.

:hankey:

**Comment by leafo** [comment]
Okay the next error is that sitegen is currently only available for 5.1. I provided implementation of the missing functions. Additionally, you're probably going to run into one more error. You'll need to install the development version of moonscript.

Sorry it's so complicated, there are a couple things I want to work out before I released a versioned release.

**Comment by leafo** [comment]
Updated luarocks.org to make it so sitegen's rockspec is linked to repo version, also updated install instructions on the module page for dev-only modules to make it more clear how to install: 

https://luarocks.org/modules/leafo/sitegen
